datacats/cli/pull.py

```diff
@@ -12,15 +12,14 @@
 
 IMAGES = [
     'datacats/web',
-    'datacats/ckan:2.3',
+    'datacats/ckan:2.4',
     'datacats/postgres',
     'datacats/solr'
     ]
 
 EXTRA_IMAGES = [
     'datacats/lessc',
-    'datacats/ckan:latest',
-    'datacats/ckan:2.4'
+    'datacats/ckan:latest'
```

Comment:
I wasn't sure if there was any reason to have the 2.3 version, but I added it back in case anyone needs it.